src/app/components/start-load-test-widget/start-load-test-widget.component.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+import { LoadTestConfig } from 'src/app/models/loadTestConfig';
+
+// NgRX
+/* import { SwagPaths } from '../../state/swag-paths/swag-paths.model';
+import { POPULATE_PATHS } from '../../state/swag-paths/swag-paths.actions';
+import { Store } from '@ngrx/store';
+import { Router } from '@angular/router';
+import { ParseError } from '@angular/compiler';
+*/
+
+@Component({
+  selector: 'app-start-load-test-widget',
+  templateUrl: './start-load-test-widget.component.html',
+  styleUrls: ['./start-load-test-widget.component.scss'],
+})
+export default class StartLoadTestWidgetComponent implements OnInit {
+  time = 0;
+
+  interval;
+
+  running = false;
+
+  advance = false;
+
+  click = false;
+
+  indexValue = 0;
+
+  LFC;
+
+  Msg = '';
+
+  public MsgShowError = false;
+
+  public MsgShowSuccess = false;
+
+  private Base_Url = 'http://localhost:8083/Docutest';
+
+  AdvanceForm = new FormGroup({
+    planName: new FormControl(''),
+    loops: new FormControl(''),
+    duration: new FormControl(''),
+    thread: new FormControl(''),
+    rampUp: new FormControl(''),
+    followRedirect: new FormControl(''),
+  });
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
If this line is added to disable this eslint rule simply for the `ngOnInit` method, then it would instead be preferable to remove this method and the corresponding interface if it isn't being used.